lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_firebase_test.rb

```diff
@@ -0,0 +1,187 @@
+              next if File.file? value
+
+              UI.user_error!("Invalid test APK: #{value}")
+            end
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :model,
+            description: 'The device model to run',
+            type: String,
+            verify_block: proc do |value|
+              model_names = Fastlane::FirebaseDevice.valid_model_names
+              next if model_names.include? value
+
+              UI.user_error!("Invalid Model Name: #{value}. Valid Model Names: #{model_names}")
+            end
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :version,
+            description: 'The device version to run',
+            type: Integer,
+            verify_block: proc do |value|
+              version_numbers = Fastlane::FirebaseDevice.valid_version_numbers
+              next if version_numbers.include? value
+
+              UI.user_error!("Invalid Version Number: #{value}. Valid Verison Numbers: #{version_numbers}")
+            end
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :locale,
+            description: 'The locale code to run in',
+            type: String,
+            default_value: 'en',
+            verify_block: proc do |value|
+              locale_codes = Fastlane::FirebaseDevice.valid_locales
+              next if locale_codes.include? value
+
+              UI.user_error!("Invalid Locale: #{value}. Valid Locales: #{locale_codes}")
+            end
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :orientation,
+            description: 'Which orientation to run the device in',
+            type: String,
+            default_value: 'portrait',
+            verify_block: proc do |value|
+              orientations = Fastlane::FirebaseDevice.valid_orientations
+              next if orientations.include? value
+
+              UI.user_error!("Invalid Orientation: #{value}. Valid Orientations: #{orientations}")
+            end
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :type,
+            description: 'Which type of test are we running?',
+            type: String,
+            default_value: 'instrumentation',
+            verify_block: proc do |value|
+              types = Fastlane::FirebaseTestRunner::VALID_TEST_TYPES
+              next if types.include? value
+
+              UI.user_error!("Invalid Test Type: #{value}. Valid Types: #{types}")
+            end
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :test_run_id,
+            description: 'A unique ID used to identify this test run',
+            type: String,
+            default_value: run_uuid
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :results_output_dir,
+            description: 'Where should we store the results of this test run?',
+            type: String,
+            default_value: File.join(Dir.tmpdir(), run_uuid)
```

Comment:
🧠  For such cases you should also specify that the default value is dynamic, so that the generated doc for the action indicates it via a `*` in the docs. See [here in the `ConfigItem` API](https://github.com/fastlane/fastlane/blob/master/fastlane_core/lib/fastlane_core/configuration/config_item.rb#L27-L28)
```suggestion
            default_value: File.join(Dir.tmpdir(), run_uuid),
            default_value_dynamic: true
```